### PR TITLE
feature/raic-317-calendar-redesign

### PR DESCRIPTION
**Pull request by MitchellShimko**

### CODE REVIEW

**Comment by Philip-Loeffler** [comment]
I would reach out to Riikka concerning the links on the footer. Currently the do not go anywhere. If she has specific routes that they should head to let's add them. If not, all good.

File: src/components/Header/Header.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+interface HeaderProps {
+  title?: string;
+  description?: string;
+}
+
+export const Header = ({ title, description }: HeaderProps) => {
+  return (
+    <div
+      style={{
+        backgroundImage: `url(/image/header.png)`,
```
Comment (ONeill45) [inline src/components/Header/Header.tsx:15]: Doesn't look like the filepath for the image leads anywhere

File: src/templates/calendar.tsx [hunk 2]
```diff
@@ -16,16 +21,31 @@ interface PageProps {
 function Calendar({ data }: PageProps) {
   const { page } = data;
 
+  const contentTabs: Tab[] = [
+    {
+      tabName: "Events",
```
Comment (ONeill45) [inline src/templates/calendar.tsx:26]: I'd add `cursor: pointer` on the tabs

File: src/components/Navigation/Navigation.tsx [hunk 3]
```diff
@@ -1,13 +1,14 @@
...
 import { graphql, Link, useStaticQuery } from "gatsby";
+import { AnimatePresence, motion } from "framer-motion";
 import { Container } from "@/components/Container";
 import { Logo } from "@/components/Logo";
 import { Icon } from "@/components/Icon";
 import { Breakpoints, useResizeObserver } from "@/hooks/useResizeObserver";
-import { AnimatePresence, motion } from "framer-motion";
 
 import { NavigationItemFragment } from "@/graphql/graphql-types";
+import { AnthemLogo } from "../AnthemLogo/AnthemLogo";
```
Comment (ONeill45) [inline src/components/Navigation/Navigation.tsx:11]: use the alias path `@/components/AnthemLogo` whenever possible instead of relative paths

**Review by ONeill45 — CHANGES_REQUESTED** [review]

**Review by Philip-Loeffler — APPROVED** [review]

File: src/components/Header/Header.module.css [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 27px;
+  margin-top: 40px;
+  display: flex;
+  color: #000000;
+}
+
+.headerContainer {
+  background-image: url(./header.png);
```
Comment (ONeill45) [inline src/components/Header/Header.module.css:26]: Add `background-repeat` and `background-size` properties to this: 
image

Right now the image is repeating and has a clear line break down the middle

**Pull request closed without merging**